src/VisualStudio/Core/Def/Implementation/Snippets/AbstractSnippetExpansionClient.cs

```diff
@@ -169,7 +169,20 @@ private void CleanUpEndLocation(ITrackingSpan endTrackingSpan)
                 if (lineText.Trim() == string.Empty)
                 {
                     indentCaretOnCommit = true;
-                    indentDepth = lineText.Length;
+
+                    var document = this.SubjectBuffer.CurrentSnapshot.GetOpenDocumentInCurrentContextWithChanges();
+                    if (document != null)
+                    {
+                        var optionService = document.Project.Solution.Workspace.Services.GetService<IOptionService>();
+                        var tabSize = optionService.GetOption(FormattingOptions.TabSize, document.Project.Language);
```

Comment:
Oh that should be fine, I just didn't want you trying to implement GetColumnFromLineOffset yourself since that's tricky.